src/Form.svelte

```diff
@@ -191,6 +192,10 @@
               <div class="mobile">
                 <Address compact resolve={field.resolve ?? false} address={field.value} {config} />
               </div>
+            {:else if (field.label === "Seed Host")}
+              <div>
+                <a use:link href="/seeds/{field.value}">{field.value}</a>
+              </div>
```

Comment:
This (and the Seed ID) check don't really belong here, as the field label isn't supposed to be an API. Will look into something cleaner.

Reply:
Right, maybe we can add a check for `isSeedHost` and `isSeedId`? Similar to `isAddress` and `isUrl`?
So it doesn't depend on the field label

Reply:
Okay that's a clean solution, thanks!